res/css/_animations.pcss

```diff
@@ -55,16 +75,40 @@ limitations under the License.
     }
 }
 
-@keyframes mx_ImageView_panel_keyframes {
+@keyframes visualbell {
```

Comment:
I am wondering if the current name should be renamed to something like `mx_visualbell` though there is not a component named as `Visualbell`.